src/components/Starred.jsx

```diff
@@ -0,0 +1,42 @@
+import { useSelector, useDispatch } from 'react-redux'
+import { Link } from 'react-router-dom'
+import starredSlice from '../data/starredSlice'
+import Movie from './Movie'
+import '../styles/starred.scss'
+
+const Starred = ({viewTrailer}) => {
+
+    const state = useSelector((state) => state)
```

Comment:
It can invoke unnecessary re-renders, I suppose we should use selectors to return the specific parts of the state that the component needs